tests/bricks/test_conv.py

```diff
@@ -259,14 +229,14 @@ def test_convolutional_sequence():
     batch_size = 5
     activation = Rectifier().apply
 
-    conv = ConvolutionalLayer(activation, (3, 3), 5,
-                              (pooling_size, pooling_size),
-                              weights_init=Constant(1.),
-                              biases_init=Constant(5.))
+    conv = ConvolutionalActivation(activation, (3, 3), 5,
```

Comment:
This is mentioned in the [docs](https://github.com/mila-udem/blocks/blob/master/blocks/bricks/conv.py#L347-L348), by the way. I'm not crazy about the name (or this class at all), but it is what it is.